src/native/main/WebCommonNativeFacade.ts

```diff
@@ -37,32 +41,54 @@ export class WebCommonNativeFacade implements CommonNativeFacade {
 			if (mailToUrlString) {
 				editor = await newMailtoUrlMailEditor(mailToUrlString, false, mailboxDetails).catch(ofClass(CancelledError, noOp))
 				if (!editor) return
+
+				editor.show()
 			} else {
 				const files = await fileApp.getFilesMetaData(filesUris)
-				const address = (addresses && addresses[0]) || ""
-				const recipients = address
-					? {
-							to: [
-								{
-									name: "",
-									address: address,
-								},
-							],
-					  }
-					: {}
-				editor = await newMailEditorFromTemplate(
-					mailboxDetails,
-					recipients,
-					subject || (files.length > 0 ? files[0].name : ""),
-					signatureModule.appendEmailSignature(text || "", logins.getUserController().props),
-					files,
-					undefined,
-					undefined,
-					true, // we want emails created in this method to always default to saving changes
-				)
-			}
+				const allFilesAreVCards = files.filter((file) => Object.values<string>(VCARD_MIME_TYPES).includes(file.mimeType)).length === files.length
+
+				let willImport = false
+				if (allFilesAreVCards) {
+					willImport = await Dialog.choice(
+						() => "We detected some contact files, do you want to import or attach them?",
```

Comment:
we should clarify the text and also not forget to translate it

Reply:
Added the keys